ckanext/unaids/plugin.py

```diff
@@ -253,6 +255,15 @@ def _giftless_upload(context, resource, current=None):
             })
 
 
+def _update_resource_last_modified_date(resource, current={}):
+    for key in ['url_type', 'lfs_prefix', 'sha256', 'size', 'url']:
+        current_value = str(current.get(key) or '')
```

Comment:
`.get()` can return custom default if key not present. 
Better:
`str(current.get(key, ''))`

Reply:
After discussion on meets we've decided that it's wrong suggestion because:
```
>>> str(None)
'None'
>>> str({'key': None}.get('key', ''))
'None'
>>> str({'key': None}.get('key') or '')
''
```